website/docs/cloud-docs/api-docs/no-code-provisioning.mdx

```diff
@@ -487,3 +487,417 @@ curl \
+      "vcs-repo-identifier": null,
+      "permissions": {
+        "can-update": true,
+        "can-destroy": true,
+        "can-queue-run": true,
+        "can-read-variable": true,
+        "can-update-variable": true,
+        "can-read-state-versions": true,
+        "can-read-state-outputs": true,
+        "can-create-state-versions": true,
+        "can-queue-apply": true,
+        "can-lock": true,
+        "can-unlock": true,
+        "can-force-unlock": true,
+        "can-read-settings": true,
+        "can-manage-tags": true,
+        "can-manage-run-tasks": true,
+        "can-force-delete": true,
+        "can-manage-assessments": true,
+        "can-manage-ephemeral-workspaces": true,
+        "can-read-assessment-results": true,
+        "can-queue-destroy": true
+      },
+      "actions": {
+        "is-destroyable": true
+      },
+      "description": null,
+      "file-triggers-enabled": true,
+      "trigger-prefixes": [],
+      "trigger-patterns": [],
+      "assessments-enabled": false,
+      "last-assessment-result-at": null,
+      "source": "tfe-module",
+      "source-name": null,
+      "source-url": null,
+      "source-module-id": "private/hashicorp/lambda/aws/1.0.9",
+      "no-code-upgrade-available": false,
+      "tag-names": [],
+      "setting-overwrites": {
+        "execution-mode": false,
+        "agent-pool": false
+      }
+    },
+    "relationships": {
+      "organization": {
+        "data": {
+          "id": "hashicorp",
+          "type": "organizations"
+        }
+      },
+      "current-run": {
+        "data": null
+      },
+      "latest-run": {
+        "data": null
+      },
+      "outputs": {
+        "data": []
+      },
+      "remote-state-consumers": {
+        "links": {
+          "related": "/api/v2/workspaces/ws-qACTToFUM5BvDKhC/relationships/remote-state-consumers"
+        }
+      },
+      "current-state-version": {
+        "data": null
+      },
+      "current-configuration-version": {
+        "data": {
+          "id": "cv-vizi2p3mnrt3utgA",
+          "type": "configuration-versions"
+        },
+        "links": {
+          "related": "/api/v2/configuration-versions/cv-vizi2p3mnrt3utgA"
+        }
+      },
+      "agent-pool": {
+        "data": null
+      },
+      "readme": {
+        "data": null
+      },
+      "project": {
+        "data": {
+          "id": "prj-yuEN6sJVra5t6XVy",
+          "type": "projects"
+        }
+      },
+      "current-assessment-result": {
+        "data": null
+      },
+      "no-code-module-version": {
+        "data": {
+          "id": "nocodever-vFcQjZLs3ZHTe4TU",
+          "type": "no-code-module-versions"
+        }
+      },
+      "vars": {
+        "data": []
+      }
+    },
+    "links": {
+      "self": "/api/v2/organizations/hashicorp/workspaces/no-code-workspace",
+      "self-html": "/app/hashicorp/workspaces/no-code-workspace"
+    }
+  }
+}
+```
+
+## Update a Workspace to use a new No-Code module version
```

Comment:
I believe it might be more consistent with other API documentation to break out a heading for each of the three steps here, rather than documenting three APIs under a single heading. I think that the descriptions of each API can handle covering where you are in the overall upgrade process, and link to the next step

Reply:
```suggestion
## Initiate a Workspace Update to new No-Code provisioning settings
```